.github/workflows/AtlasReleaseThemeFiles.yml

```diff
@@ -46,11 +46,9 @@ jobs:
                   source: "packages/theming/atlas/src/theme/web/*"
                   target: "packages/theming/atlas/dist/theme/web/"
             - name: "Create ZIP"
-              uses: papeloto/action-zip@v1
+              uses: montudor/action-zip@v1
```

Comment:
This incident shows that it's not safe enough to target tags of external actions. We should use full commit sha's instead. See also:
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
Fixed for all custom gh actions. Left only official from github with tags